keps/sig-instrumentation/0034-distributed-tracing-kep.md

```diff
@@ -255,42 +146,44 @@ GA
     How many objects of that type will be supported (and how that translates
     to limitations for users)?
   - Will enabling / using the feature result in increasing size or count
-    of the existing API objects?  **Yes.  It adds an annotation to "traced" objects.  The value is a trace context, which is ~32 bytes.  Traced objects will initially include pods, replicasets, and deployments, but may expand to include others over time.  Notably, this annotation should not be added to Events.**
+    of the existing API objects?  **No.**
   - Will enabling / using the feature result in increasing time taken
     by any operations covered by [existing SLIs/SLOs][] (e.g. by adding
-    additional work, introducing new steps in between, etc.)? **No**
+    additional work, introducing new steps in between, etc.)? **Yes.  It will increase API Server request latency by a negligible amount**
```

Comment:
Can you clarify this one?
Is the sending to tracing backend on the critical path? Does it have to be (can't it be done asynchronously)?

Reply:
Sending spans to the tracing backend is not on the critical path.  Decoding the trace context from headers, recording the span in memory, and encoding the trace context into headers for requests to clients are on the critical path, and each can add up to 1 microsecond of latency based on benchmarks (e.g. https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/trace/benchmark_test.go, https://github.com/open-telemetry/opentelemetry-go/pull/194).

Reply:
OK - that sounds fine. Can you please add that to the doc FTR (the explanation is really useful).

Reply:
Updated.